aries-site/src/data/wcag/datatable.json

```diff
@@ -0,0 +1,518 @@
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.4",
+    "label": "Captions (Live)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.5",
+    "label": "Audio Description (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.6",
+    "label": "Sign Language (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.7",
+    "label": "Extended Audio Description (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.8",
+    "label": "Media Alternative (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.9",
+    "label": "Audio-only (Live)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.1",
+    "label": "Info and Relationships",
+    "status": "failed",
+    "notes": "sortable and searchable headings need improvement"
+  },
+  {
+    "rule": "1.3.2",
+    "label": "Meaningful Sequence",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.3",
+    "label": "Sensory Characteristics",
+    "status": "failed",
+    "notes": "sortable and searchable headings need improvement"
+  },
+  {
+    "rule": "1.3.4",
+    "label": "Orientation",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.5",
+    "label": "Identify Input Purpose",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.6",
+    "label": "Identify Purpose",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.1",
+    "label": "Use of Color",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.2",
+    "label": "Audio Control",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.3",
+    "label": "Contrast (Minimum)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.4",
+    "label": "Resize Text",
+    "status": "conditional",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.5",
+    "label": "Images of Text",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.6",
+    "label": "Contrast (Enhanced)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.7",
+    "label": "Low or No Background Audio",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.8",
+    "label": "Visual Presentation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.9",
+    "label": "Images of Text (No Exception)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.4.10",
+    "label": "Reflow",
+    "status": "conditional",
+    "notes": "This really depends on content but the developer almost always must switch representations or hide columns at smaller sizes."
+  },
+  {
+    "rule": "1.4.11",
+    "label": "Non-text Contrast",
+    "status": "passed",
```

Comment:
I don't think we are passing this, for rows that are selectable we indicate which row is in focus via a grey background that I don't think meets 3:1

Reply:
would pinned be considered more a decorative item? We weren't sure how `pinned` would help someone who visually could not see the benefits of this prop

Reply:
Ill check this 

> I don't think we are passing this, for rows that are selectable we indicate which row is in focus via a grey background that I don't think meets 3:1

Reply:
Pinned columns don't normally have a gray backround. That's just something the MultiplePins story is doing. Normally a pinned column simply has a white (plus maybe a blur) background

Reply:
okay in this story example we would not be passing here for rows that are selectable https://storybook.grommet.io/?path=/story/visualizations-datatable-clickable--clickable-data-table&globals=theme:hpe

changed ruling and added note

Reply:
I like its more for the selectable because when you tab to the whole datatable there is a focus indicator around datatable but when you arrow through you just visually see the grey hover color which isnt passing color contrast is how I interpreted this

Reply:
I feel like it would be helpful to have some accessible indication of a pinned column so that a user understands why it is 'sticky' and in the case where we support column re-ordering they understand why a pinned column can't be re-ordered

Reply:
> We weren't sure how pinned would help someone who visually could not see the benefits of this prop

I think it would be helpful for people who can see but can't detect the color difference between pinned vs non-pinned column

Reply:
I would think this would be contrast of text on a background rather than graphics on a background so 1.4.3 I'd think given the descriptions

Reply:
> I feel like it would be helpful to have some accessible indication of a pinned column so that a user understands why it is 'sticky' and in the case where we support column re-ordering they understand why a pinned column can't be re-ordered

So maybe it needs that status in the header aria label similar to the sort direction being in there. Seems like an annoying thing to have read to you when you can't see it though

Reply:
> > We weren't sure how pinned would help someone who visually could not see the benefits of this prop
> 
> I think it would be helpful for people who can see but can't detect the color difference between pinned vs non-pinned column

There is no color difference in our theme today

Reply:
Ah okay if we don't indicate pinned visually via the theme then I don't think we need an accessible indication for it